<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeaderDialog</class>
 <widget class="QWidget" name="LeaderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>23</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(252, 42, 8);
font: 23pt &quot;Futura&quot;;</string>
   </property>
   <property name="text">
    <string>Leader Board</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 102);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="num_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="num_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>95</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="num_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="num_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>145</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="num_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="num_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>195</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="num_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="num_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>245</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="num_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 102);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>75</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 0);color: rgb(255, 153, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>125</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>175</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>225</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>275</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score_1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>75</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>125</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>175</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>225</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score_9">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>275</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 153, 153);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="resume">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>295</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(102, 153, 255);</string>
   </property>
   <property name="text">
    <string>resume</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
